test/parallel/test-util-deprecate-invalid-code.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+const common = require("../common");
+const util = require("util");
+
+[1, true, false, null, {}].forEach(notString => {
+  common.expectsError(
+    () => {
+      util.deprecate(() => {}, "message", 123);
+    },
```

Comment:
nit: There's only one statement inside the function.
This can be converted to:
```javascript
() => util.deprecate(() => {}, "message", 123)
```

Reply:
Done and updated